backend/routes.js

```diff
@@ -7,40 +7,164 @@ const mongoose = require('mongoose');
+    skinConcerns: ["acne", "blackheads"],
+    isSustainable: true
+  },
+  {
+    id: 6,
+    name: "hyaluronic acid hydrating serum",
+    brand: "the inkey list",
+    price: 19.99,
+    rating: 4.4,
+    description: "Multi-molecular weight hyaluronic acid for deep hydration.",
+    ingredient_list: ["hyaluronic acid", "sodium hyaluronate", "glycerin", "water", "panthenol"],
+    skinTypes: ["dry", "sensitive"],
+    skinConcerns: ["dryness", "dehydration"],
+    isSustainable: true
+  },
+  {
+    id: 7,
+    name: "gentle exfoliating toner",
+    brand: "pixi",
+    price: 28.00,
+    rating: 4.2,
+    description: "Gentle glycolic acid toner for smooth, radiant skin.",
+    ingredient_list: ["glycolic acid", "aloe vera", "ginseng", "water", "witch hazel"],
+    skinTypes: ["normal", "combination"],
+    skinConcerns: ["dullness", "texture"],
+    isSustainable: false
+  },
+  {
+    id: 8,
+    name: "ceramide repair moisturizer",
+    brand: "cerave",
+    price: 26.99,
+    rating: 4.5,
+    description: "Rich moisturizer with ceramides for barrier repair.",
+    ingredient_list: ["ceramides", "cholesterol", "fatty acids", "hyaluronic acid", "dimethicone"],
+    skinTypes: ["dry", "sensitive"],
+    skinConcerns: ["dryness", "barrier damage"],
+    isSustainable: true
+  },
+  {
+    id: 9,
+    name: "zinc sunscreen spf 50",
+    brand: "eltamd",
+    price: 41.00,
+    rating: 4.8,
+    description: "Broad spectrum mineral sunscreen with zinc oxide.",
+    ingredient_list: ["zinc oxide", "titanium dioxide", "octinoxate", "water", "silica"],
+    skinTypes: ["all"],
+    skinConcerns: ["sun protection"],
+    isSustainable: false
+  },
+  {
+    id: 10,
+    name: "peptide firming cream",
+    brand: "olay",
+    price: 37.99,
+    rating: 4.3,
+    description: "Anti-aging cream with peptides and amino acids.",
+    ingredient_list: ["peptides", "amino acids", "niacinamide", "glycerin", "dimethicone"],
+    skinTypes: ["normal", "dry"],
+    skinConcerns: ["aging", "firmness"],
+    isSustainable: false
+  },
+  {
+    id: 11,
+    name: "tea tree oil spot treatment",
+    brand: "the body shop",
+    price: 15.00,
+    rating: 4.1,
+    description: "Targeted acne treatment with tea tree oil.",
+    ingredient_list: ["tea tree oil", "salicylic acid", "witch hazel", "alcohol", "water"],
+    skinTypes: ["oily", "acne-prone"],
+    skinConcerns: ["acne", "blemishes"],
+    isSustainable: true
+  },
+  {
+    id: 12,
+    name: "rose hip oil facial oil",
+    brand: "trilogy",
+    price: 29.99,
+    rating: 4.6,
+    description: "Pure rosehip oil for hydration and anti-aging.",
+    ingredient_list: ["rosehip seed oil", "vitamin e", "linoleic acid", "oleic acid", "palmitic acid"],
+    skinTypes: ["dry", "mature"],
+    skinConcerns: ["aging", "dryness"],
     isSustainable: true
   }
 ];
 
+const mockIngredients = [
+  { id: 1, name: "hyaluronic acid", benefits: ["hydration", "plumping"], category: "humectant" },
```

Comment:
why do we need to mock ingredients? Let's get this from an API. Mocking should be done mainly for testing purposes